src/accumulator.jl

```diff
@@ -4,12 +4,11 @@ type Accumulator{T, V<:Number}
     map::Dict{T,V}
 end
 
-## constructors
+# Constructors
```

Comment:
I don't care too much for these comments, but if they're present, they should be consistent. If we're going to go through the trouble to change this one, can we also change the others (`## retrieval`, `## iteration`, etc)?